vm/systemSmartContracts/auction_test.go

```diff
@@ -48,6 +50,80 @@ func createABid(totalStakeValue uint64, numBlsKeys uint32, maxStakePerNode uint6
 	return data
 }
 
+func TestStakingAuctionSmartContract_NilValidatorSetting(t *testing.T) {
```

Comment:
Test**New**Staking... ? also in the tests below

Reply:
done.